variables/Four Column Lookup.tpl

```diff
@@ -0,0 +1,306 @@
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertUndefined",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertUndefinedTo",
+        "type": "TEXT"
+      },
+      {
+        "simpleValueType": true,
+        "name": "convertTrue",
+        "checkboxText": "Convert true to…",
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertTrue",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertTrueTo",
+        "type": "TEXT"
+      },
+      {
+        "simpleValueType": true,
+        "name": "convertFalse",
+        "checkboxText": "Convert false to…",
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertFalse",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertFalseTo",
+        "type": "TEXT"
+      }
+    ]
+  }
+]
+
+
+___WEB_PERMISSIONS___
+
+[
+  {
+    "instance": {
+      "key": {
+        "publicId": "logging",
+        "versionId": "1"
+      },
+      "param": [
+        {
+          "key": "environments",
+          "value": {
+            "type": 1,
+            "string": "debug"
+          }
+        }
+      ]
+    },
+    "isRequired": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const makeString = require('makeString');
+const log = require('logToConsole');
+const inputVar = makeString(data.inputVar);
+const table = data.lookupTable;
+const defaultValue = data.defaultValue;
+//Format Value Values
+const convertCaseTo = data.convertCaseTo;
+const convertCase = data.convertCase;
+const convertNullTo = data.convertNullTo;
+const convertNull = data.convertNull;
+const convertUndefinedTo = data.convertUndefinedTo;
+const convertUndefined = data.convertUndefined;
+const convertTrueTo = data.convertTrueTo;
+const convertTrue = data.convertTrue;
+const convertFalseTo = data.convertFalseTo;
+const convertFalse = data.convertFalse;
+//Declare Variables
+//If you need to change your default delimter, do it here.
+var val = defaultValue !== undefined ? defaultValue.split('|') : undefined,
+    obj = [],
+    a,
+    b;
+
+if(typeof(table) !=='undefined'){
+table.forEach(function(row){
+  var arr = {};
+  arr[row.inputVal]=[row.output1,row.output2,row.output3];
+  obj.push(arr);
+});
+}
+for(a=0;a<obj.length;a++){
+if (typeof(obj[a][inputVar]) !== 'undefined'){
+  var val = obj[a][inputVar];
+  break;
+} 
+}
+
+if (val !== undefined){
+  //Convert variables
+  for(a=0;a<val.length;a++){
+    if(convertUndefined === true && val[a]== 'undefined'){
```

Comment:
These checks would only work if the value is actually a string "undefined", "null", "true", etc. Is that the purpose?

If you need to check for actually undefined, you need to do 

`val[a] === undefined`
`val[a] === null`
`val[a] === true`
`val[a] === false`